src/System.Private.CoreLib/src/System/Threading/Thread.CoreRT.Windows.cs

```diff
@@ -30,6 +30,7 @@ public sealed partial class Thread
 
         private void PlatformSpecificInitialize()
         {
+            RuntimeImports.RhSetThreadExitCallback(AddrofIntrinsics.AddrOf<Action>(OnThreadExit));
```

Comment:
Ah, this one could be moved to the common file as well. Or do we want to keep interop things in their platform files even when there is identical helper code calling native functions?

Reply:
This is interop with the runtime itself, not with the OS-specific APIs. We prefer to keep interop like that in the platform neutral files. It is usually same accross all OSes.